docs/wxm-devices/wifi-m5-bundle/m5-connect-and-claim.mdx

```diff
@@ -5,19 +5,36 @@ sidebar_label: Connect and Claim Station
 slug: /wxm-devices/wifi-m5-bundle/m5-connect-and-claim
 ---
 
+import Tabs from "@theme/Tabs";
 import useBaseUrl from "@docusaurus/useBaseUrl";
 
+## M5 Miner
+<Tabs groupId="wg1000_ui">
+  <TabItem value="wg1000_ui_new" label="UI versions > 0.10.0">
+  <img object-fit="contain" src={useBaseUrl("img/wxm-devices/wifi-m5-bundle/m5-ui-new.svg")} />
```

Comment:
On my (local) computer the image appears with a few sec delay. Possibly trying to render all its path elements